app/helpers/ndr_ui/bootstrap_helper.rb

```diff
@@ -243,6 +243,34 @@ def bootstrap_form_for(record_or_name_or_array, *args, &proc)
       end
     end
 
+    # Identical signature to form_with, but uses NdrUi::BootstrapBuilder.
```

Comment:
If we're going to say the signature is identical, can we make it identical?

```ruby
# lib/action_view/helpers/form_helper.rb
def form_with(model: nil, scope: nil, url: nil, format: nil, **options)
```

Or perhaps more sensibly, use the double-splat operator:

```ruby
def bootstrap_form_with(**options, &block)
  # ... customise `options` ...
  form_with(**options, &block)
end